ui/src/org/pentaho/di/ui/trans/steps/fileinput/text/TextFileInputDialog.java

```diff
@@ -1491,20 +1495,41 @@ public void focusGained( org.eclipse.swt.events.FocusEvent e ) {
       }
     } );
 
+    wlLength = new Label( wContentComp, SWT.RIGHT );
+    wlLength.setText( BaseMessages.getString( PKG, "TextFileInputDialog.Length.Label" ) );
+    props.setLook( wlLength );
+    fdlLength = new FormData();
+    fdlLength.left = new FormAttachment( 0, 0 );
+    fdlLength.top = new FormAttachment( wEncoding, margin );
+    fdlLength.right = new FormAttachment( middle, -margin );
+    wlLength.setLayoutData( fdlLength );
+    wLength = new CCombo( wContentComp, SWT.BORDER | SWT.READ_ONLY );
+    wLength.setText( BaseMessages.getString( PKG, "TextFileInputDialog.Length.Label" ) );
+    props.setLook( wLength );
+    wLength.add( "Characters" );
```

Comment:
@HiromuHota

We missed this in the first review, but there is also a need to have the "Characters" and "Bytes" strings defined by message properties, so that it can be localized into other languages.  As an example, the [CheckSumDialog class](https://github.com/pentaho/pentaho-kettle/blob/6.1.0.1-R/ui/src/org/pentaho/di/ui/trans/steps/checksum/CheckSumDialog.java#L155-L160) does this for selecting the CheckSum Type.